### ISSUE TITLE
show and tell variables

### ISSUE THREAD

**Issue by sweaterr** [sweaterr/deep-learning-readinglist#85]
입력 
image_feed
input_feed

모델
lstm/initial_state
lstm/state_feed
lstm/state

출력
logit
softmax
batch_loss
